interface/patient_file/front_payment.php

```diff
@@ -401,6 +401,11 @@ function calcTaxes($row, $amount)
     WHERE encounter = ?", array($payrow['encounter']));
     $frow = $facilityService->getById($tmprow['facility_id']);
 
+ if($_REQUEST['radio_type_of_payment']=='pre_payment'){
+  // if pre_payment is being made then use the main office address added by Sherwin 7/21/2017
+   $frow = sqlQuery("SELECT * FROM `facility` WHERE name != '' LIMIT 1");
+   }
```

Comment:
Is there a bug in the above code?:
``````
$frow = $facilityService->getById($tmprow['facility_id']);
``````
We should fix that bug rather than go around it with another sql query.